integration-cli/docker_api_containers_test.go

```diff
@@ -650,7 +650,7 @@ func (s *DockerSuite) TestContainerApiCommit(c *check.C) {
 	// sanity check, make sure the image is what we think it is
 	out, err = exec.Command(dockerBinary, "run", img.Id, "ls", "/test").CombinedOutput()
 	if err != nil {
-		c.Fatal(out, err)
+		c.Fatal(string(out), err)
```

Comment:
Since you're touching this line anyway... perhaps it would be more friendly to someone looking at the output if it were:
```
c.Fatal("run failed. out:%s\nerr:%s", out, err)
```
Not a fan of just printing variables like 'err' w/o some text around it to give context.